plugins/modules/vmware_guest_vgpu.py

```diff
@@ -0,0 +1,362 @@
+        """
+        vm_current_vgpu_profile = []
+        for vgpu_VirtualDevice_obj in vm_obj.config.hardware.device:
+            if hasattr(vgpu_VirtualDevice_obj.backing, 'vgpu'):
+                vm_current_vgpu_profile.append(vgpu_VirtualDevice_obj.backing.vgpu)
+        return vm_current_vgpu_profile
+
+    def _get_vgpu_VirtualDevice_object(self, vm_obj, vgpu_prfl):
+        """
+        Get current vGPU profile object of virtual machine
+        Args:
+            vm_obj: Managed object of virtual machine
+            vgpu_prfl: vGPU profile name
+        Returns: vGPU profile name of virtual machine
+        """
+        for vgpu_VirtualDevice_obj in vm_obj.config.hardware.device:
+            if hasattr(vgpu_VirtualDevice_obj.backing, 'vgpu'):
+                if vgpu_VirtualDevice_obj.backing.vgpu == vgpu_prfl:
+                    return vgpu_VirtualDevice_obj
+                    break
+        return None
+
+    def _get_vgpu_profiles_name(self, vm_obj, vgpu_prfl):
+        """
+        Get matched vGPU profile object of ESXi host
+        Args:
+            vm_obj: Managed object of virtual machine
+            vgpu_prfl: vGPU profile name
+        Returns: vGPU profile object
+        """
+        vm_host = vm_obj.runtime.host
+        vgpu_profiles = vm_host.config.sharedGpuCapabilities
+        for vgpu_profile_name in vgpu_profiles:
+            if vgpu_profile_name.vgpu == vgpu_prfl:
+                return vgpu_profile_name
+                break
+        return None
+
+
+def main():
+    argument_spec = vmware_argument_spec()
+    argument_spec.update(
+        name=dict(type='str'),
+        uuid=dict(type='str'),
+        use_instance_uuid=dict(type='bool', default=False),
+        moid=dict(type='str'),
+        folder=dict(type='str'),
+        datacenter=dict(type='str', default='ha-datacenter'),
+        esxi_hostname=dict(type='str'),
+        cluster=dict(type='str'),
+        vgpu=dict(type='str'),
+        force=dict(type='bool', default=False),
+        state=dict(type='str', default='present', choices=['absent', 'present'])
+    )
+
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        mutually_exclusive=[
+            ['cluster', 'esxi_hostname']
+        ],
+        required_one_of=[
+            ['name', 'uuid', 'moid']
+        ],
+    )
+
+    pyv = PyVmomiHelper(module)
+    vm = pyv.get_vm()
+
+    if not vm:
+        vm_id = (module.params.get('uuid') or module.params.get('name') or module.params.get('moid'))
+        module.fail_json(msg="Unable to manage vGPU profile for non-existing VM {}".format(vm_id))
+
+    if module.params['state'] == 'present':
+        result = pyv._vgpu_present(vm)
+    elif module.params['state'] == 'absent':
+        result = pyv._vgpu_absent(vm)
+
+    if 'failed' not in result:
+        result['failed'] = False
+
+    if result['failed']:
+        module.fail_json(**result)
+    else:
+        module.exit_json(**result)
+
+
```

Comment:
```suggestion
plugins/modules/vmware_guest_vgpu.py compile-2.6!skip
plugins/modules/vmware_guest_vgpu.py import-2.6!skip
```
Add these two lines in 

- https://github.com/ansible-collections/community.vmware/blob/main/tests/sanity/ignore-2.9.txt
- https://github.com/ansible-collections/community.vmware/blob/main/tests/sanity/ignore-2.10.txt
- https://github.com/ansible-collections/community.vmware/blob/main/tests/sanity/ignore-2.11.txt

Reply:
This will fix CI error.